<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaveProperty</class>
 <widget class="QDialog" name="WaveProperty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>波形图属性</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <widget class="QWidget" name="tabWidgetPage1">
      <attribute name="title">
       <string>波形图</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>图形颜色</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>名称区域:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_btnNameColor">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>相对时标零点:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_btnFaultColor">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>绘图区域:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_btnBackColor">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>△t起点:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_btnZeroColor">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>游      标:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_btnCusorColor">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>其他</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>波形线宽:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="m_edtLineWidth"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>开关量跃变形状:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="m_cbbDigtalKind"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>游标快速移动步长:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="m_cbbCusorStep"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>横向比例微调系数:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="m_edtHorzZoom"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="m_chkAutoSet">
              <property name="text">
               <string>曲线隐藏时自动分配波形区域</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>显示选项:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="m_lvOptions"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_btnHelp">
       <property name="text">
        <string>帮助(&amp;H)</string>
       </property>
       <property name="shortcut">
        <string>Alt+H</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnOk">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnCancel">
       <property name="text">
        <string>取消</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
